modules/lsp/src/main/scala/playground/lsp/ModelLoader.scala

```diff
@@ -56,28 +58,12 @@ object ModelLoader {
   ): Model = Model
     .assembler()
     .putProperty(ModelAssembler.DISABLE_JAR_CACHE, true)
-    .pipe(addModelsFromJars(classLoader.getURLs().map(_.toURI()).map(Paths.get(_).toFile())))
+    .discoverModels(classLoader)
     .pipe(addPlaygroundModels(this.getClass().getClassLoader()))
     .pipe(addFileImports(specs))
     .assemble()
     .unwrap()
```

Comment:
Still missing: a test case that fails before this change. I think it'd have to include an attempt to extract a manifest from a file with url-encoded characters in the name.